client/app/src/main/res/navigation/app.xml

```diff
@@ -120,7 +120,15 @@
         android:id="@+id/dashboardActivity"
         android:name="com.microsoft.research.karya.ui.dashboard.DashboardFragment"
         android:label="activity_dashboard"
-        tools:layout="@layout/fragment_dashboard" />
+        tools:layout="@layout/fragment_dashboard" >
+        <action
+            android:id="@+id/action_dashboardActivity_to_speechDataMainFragment2"
+            app:destination="@id/speechDataMainFragment2" >
+            <argument
+                android:name="taskId"
+                android:defaultValue="null" />
```

Comment:
Is the null `taskId` handled in the fragment? I beleive we should keep it non-null so that the user must provide the `taskId` to the fragment.

Reply:
A null value should never happen and would crash the app